detection_rules/schemas/definitions.py

```diff
@@ -32,7 +33,7 @@
     "allow_sample": (Version.parse('8.6.0'), None),
     "elasticsearch_validate_optional_fields": (Version.parse('7.16.0'), None)
 }
-NON_DATASET_PACKAGES = ['apm', 'endpoint', 'system', 'windows', 'cloud_defend', 'network_traffic']
+NON_DATASET_PACKAGES = ['apm', 'endpoint', 'system', 'windows', 'cloud_defend', 'network_traffic', 'lmd']
```

Comment:
Will this list grow to basically be NON_DATASET_PACKAGES + ANALYTIC_PACKAGES?

Reply:
@Mikaayenson yes I was thinking about this.

The `NON_DATASET_PACKAGES` are packages that are the following:

- valid packages with a manifest
- have policy templates that match the package name
- no separate integrations

The `ANALYTIC_PACKAGES` are specifically packages by the ML team only.

- valid packages with a manifest
- have no policy template
- no separate integrations

Code logic wise, `ANALYTIC_PACKAGES` is referenced in unit testing as these are exceptions for the rule tags and index requirements defined in the rule. We now have rules that may have `lmd` as a tag, but no direct correlation based on index defined and thus will fail unit testing.

`NON_DATASET_PACKAGES`, are used as an exception when adding `related_integrations` data to the rule during API conversion.

Reply:
combine lists for analytic package and non dataset when doing logic for related integrations field.